src/matrix_cookbook/3_inverses.lean

```diff
@@ -57,27 +34,9 @@ lemma eq_148 : adjugate A = (adjugate A)ᵀᵀ := rfl
 lemma eq_149 (n : ℕ) (A : matrix (fin n.succ) (fin n.succ) ℂ) :
   det A = ∑ i : fin n.succ, (-1) ^ (i : ℕ) * A i 0 * det (A.submatrix i.succ_above fin.succ) :=
 det_succ_column_zero _
-lemma eq_150 (n : ℕ) (A : matrix (fin n.succ) (fin n.succ) ℂ) : 
-  -- (-1) ^ (i : ℕ) * adjugate A 0 i represents cofactor
```

Comment:
I already established in the previous equations that `adjugate A j i` was the spelling of `Cof(A, i, j)`

Reply:
I assume you mean equation 149 right?
I think it is better to put the definition of cofactor somewhere and use it. If I understand correctly you are "mentally replacing" `cofactor A i j` everywhere with `adjugate A j i` then doing the proofs. This is of course valid but makes the code look "strained" and leads to the statement that is true about any matrix not only `adjugate A`
`lemma eq_148 : adjugate A = (adjugate A)ᵀᵀ := rfl`

Reply:
Whatever definition you pick, you're going to end up with one proof that's true by `rfl`. Pragmatically there's very little point adding `cofactor A i j` to mathlib if we already have `adjugate A j i`. It doesn't matter that they're defined in different ways; if they're mathematically equal then it's best to have one definition and a lemma that rewrites it to the other implementation.